src/test/java/net/unicon/lti/security/lti/LTI3OAuthProviderProcessingFilterTest.java

```diff
@@ -24,6 +24,7 @@
 import java.security.GeneralSecurityException;
 import java.security.KeyPair;
 import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
```

Comment:
I don't think you're catching this anymore, are you (should be a child of GeneralSecurityException)?  Probably wise to remove the unnecessary import now if it is unnecessary.